CHANGELOG.md

```diff
@@ -91,6 +91,10 @@ stabilized.
   ([#478](https://github.com/open-telemetry/semantic-conventions/pull/478))
 - Remove outdated `http.request.header.host` guidance
   ([#479](https://github.com/open-telemetry/semantic-conventions/pull/479))
+- Rename `system.processes.*` namespace to `system.process.*`
+  ([#484](https://github.com/open-telemetry/semantic-conventions/pull/484))
+- Rename `system.processes.created` to `system.process.created`
+  ([#484](https://github.com/open-telemetry/semantic-conventions/pull/484))
```

Comment:
I guess this can already be inferred by the entry above, about renaming the entire namespace?

Reply:
Good catch, thank's!